zio-schema/shared/src/main/scala/zio/schema/validation/Regex.scala

```diff
@@ -82,6 +83,19 @@ object Regex {
 
   def oneOf(chars: Char*): Regex = CharacterSet(chars.toSet)
 
+  def literalCharacter(char: Char): Regex = CharacterSet(Set(char))
+
+  def concatenation(str: String): Regex = {
```

Comment:
I would call this `literal` because you are looking for a regular expression that matches a literal string.

Also please add in alphabetical order!

Reply:
done, changed it accordingly